.travis.yml

```diff
@@ -11,7 +11,7 @@ env:
     - DEPENDENCIES="--prefer-lowest --prefer-stable"
 
 before_script:
-  - composer update --prefer-source $DEPENDENCIES
+  - composer update --prefer-dist $DEPENDENCIES
```

Comment:
Nope, GitHub changed their API limits, so this shouldn't be necessary any more, see: composer/composer#4884